content/en/guides/features/deployment-targets.md

```diff
@@ -49,7 +36,7 @@ We are also exposing `process.target` for module authors to add logic depending
 
 </base-alert>
 
-## Server Hosting
+## Server Rendering
 
 Server Hosting is hosting that requires a server and is intended for SSR applications. Server-side rendering otherwise known as SSR means that your page is rendered on the server when it is requested by the user. When the user opens your page in a browser the browser sends a request to the server requesting that page. The page is rendered on the server and sent back to the browser with all its content.
 
```

Comment:
for ssr or for using an API with server middlewares
target: server which is default
ssr: true which is default

if (usingServerMiddleware || ssr === true) {
  use server|serverless hosting
} else {
 use static hosting
} (edited) 
Eligible for static hosting <===> not using serverMiddleware